docs/guides/basics/parameters/index.md

```diff
@@ -72,6 +72,67 @@ The order that you define the parameters determines the order they are passed in
+    The generated HTML for a given value is not a stable API surface that you should rely on. We may change it at any time without warning, so we don't recommend that you parse it to extract information. Use it for display purposes only.
+
+
+### Numbers
+
+Use the `Number` parameter type to pass a number to your formula. Coda will automatically apply the [`ToNumber()`][ToNumber] formula to the input and pass it to the `execute` function as a [JavaScript Number][mdn_number].
+
+The number equivalent for some column types may not be obvious. Specifically:
+
+- **Percent** values will be converted into the equivalent fraction. For example, "75%" will be passed as `0.75`.
+- **Date** and **Date and time** values will be converted into the number of days since 1899-12-30[^1]. For example, "1955-11-12" will be passed as `20405`.
+- **Time** and **Duration** values will be converted into a number of days. For example, "12 hrs" will be passed as `0.5`.
+
+
+### Booleans
+
+Use the `Boolean` parameter type to pass a boolean (true/false) to your formula. Coda will pass the value to the `execute` function as a [JavaScript Boolean][mdn_boolean].
+
+
+### Dates
+
+Use the `Date` parameter type to pass a date value to your formula. Coda will automatically apply the [`ToDateTime()`][ToDateTime] formula to the input and pass it to the `execute` function as a [JavaScript Date][mdn_date].
+
+JavaScript Date objects can only represent a specific moment at time. This means that they can't easily represent less specific concepts like a day (regardless of time), a time (regardless of day), or duration. Coda handles those column types using the following logic:
+
+- **Date** values will be converted into a datetime representing midnight on that day in the document's timezone.
+- **Time** and **Duration** values will be converted a datetime that is that much time past midnight on 1899-12-30[^1], in the document's timezone. For example, "12 hours" in a document set to UTC will be passed as `1899-12-30T12:00:00.000Z`.
+
+!!! warning "Dates and timezones"
+    Because of how timezones work in Coda and JavaScript, the date passed into the parameter may appear different in your Pack code. See the [Dates & times][dates] guide for more information.
+
+
+### Lists
+
+Each of the parameter types described above has an array variant that allows you to pass a list of values of that type. For example, `StringArray` and `NumberArray`. All of the values in the list must be of the same type.
+
+
+### Objects
+
+Pack formulas can return structured data as [Objects][data_types_objects], but it's not possible to pass those values as parameters. You can pass their properties individually using the parameter types described above, but that can get verbose if the objects are large. A more common pattern is to have a unique identifier for each object (which is required by sync tables anyway) and pass that property of the object to other formulas that need to reference it.
```

Comment:
Up to you if you think it's useful to add some rationale for why we don't support objects, since the question comes up periodically. (It's largely because objects aren't really a concept in the Coda Formula Language and users don't have a direct way to construct them, so they would make a pack very hard for users.)

I didn't quite follow the piece about passing object ids instead of objects; I've gathered that the common desire is to pass a big bundle of input data, so I didn't make the connection for how an object id is a substitute for that.

Reply:
Done, PTAL. I tried to clarify that passage with an example.